specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/subAssessments.json

```diff
@@ -0,0 +1,604 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Security Center",
+		"description": "API spec for Microsoft.Security (Azure Security Center) resource provider",
+		"version": "2019-01-01-preview"
+	},
+	"host": "management.azure.com",
+	"schemes": ["https"],
+	"consumes": ["application/json"],
+	"produces": ["application/json"],
+	"security": [{
+		"azure_auth": ["user_impersonation"]
+	}],
+	"securityDefinitions": {
+		"azure_auth": {
+			"type": "oauth2",
+			"authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+			"flow": "implicit",
+			"description": "Azure Active Directory OAuth2 Flow",
+			"scopes": {
+				"user_impersonation": "impersonate your user account"
+			}
+		}
+	},
+	"paths": {
+		"/{scope}/providers/Microsoft.Security/assessments/{assessmentName}/subAssessments": {
+			"get": {
+				"x-ms-examples": {
+					"List security sub-assessments": {
+						"$ref": "./examples/SubAssessments/ListSubAssessments_example.json"
+					}
+				},
+				"tags": ["SubAssessments"],
+				"description": "Get security sub-assessments on all your scanned resources inside a scope",
+				"operationId": "SubAssessments_List",
+				"parameters": [
+					{
+						"$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+					},
+					{
+						"$ref": "../../../common/v1/types.json#/parameters/Scope"
+					},
+					{
+						"$ref": "#/parameters/AssessmentName"
+					},
+					{
+						"$ref": "#/parameters/ODataFilter"
+					},
+					{
+						"$ref": "#/parameters/ODataSelect"
+					},
+					{
+						"$ref": "#/parameters/ODataExpand"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "OK",
+						"schema": {
+							"$ref": "#/definitions/SecuritySubAssessmentList"
+						}
+					},
+					"default": {
+						"description": "Error response describing why the operation failed.",
+						"schema": {
+							"$ref": "../../../common/v1/types.json#/definitions/CloudError"
+						}
+					}
+				},
+				"x-ms-pageable": {
+					"nextLinkName": "nextLink"
+				}
+			}
+		},
+		"/{resourceId}/providers/Microsoft.Security/assessments/{assessmentName}/subAssessments/{subAssessmentName}": {
```

Comment:
please use same parameter as the API above. so you can call it {scope}

Reply:
@chlahav We talked about it a bit F2F.
Can you clarify?

Reply:
replace {resourceId} with  {scope} as Gaurav suggested
{
      "$ref": "../../../common/v1/types.json#/parameters/Scope"
     },

---
In reply to: [299172776](https://github.com/Azure/azure-rest-api-specs/pull/6422#discussion_r299172776) [](ancestors = 299172776)